run/browsers_test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import json
+import os
+import re
+
+
+VALID_PLATFORM_ID_REGEX = r"^[a-z0-9\-\.]+$"
+REQUIRED_PLATFORM_FIELDS = {
+    'currently_run': (True, False),
+    'initially_loaded': (True, False),
+    'browser_name': ('chrome', 'firefox', 'edge'),
+    'browser_version': None,
+    'os_name': ('linux', 'windows'),
+    'os_version': None,
+}
+
+class TestBrowsers(unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestBrowsers, self).__init__(*args, **kwargs)
+        filepath = os.path.join(os.path.dirname(__file__), os.pardir, 'browsers.json')
+        with open(filepath) as f:
+            self.browsers = json.load(f)
```

Comment:
Ah that works, some (probably unreliable) source led be to believe that functionality was a Python 3+ thing